ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -54,6 +55,12 @@ const SystemHealthDashboardPage = () => {
                     <GridItem span={12}>
                         <DeclarativeConfigurationHealthCard pollingCount={pollingCountFaster} />
                     </GridItem>
+                    <GridItem span={12}>
+                        <CertificateCard component="CENTRAL" pollingCount={pollingCountSlower} />
+                    </GridItem>
+                    <GridItem span={12}>
+                        <CertificateCard component="SCANNER" pollingCount={pollingCountSlower} />
+                    </GridItem>
```

Comment:
@vjwilson
* Do we need conditional rendering to omit for cloud service?
* If yes, remind me how we are doing that?

Reply:
1. Yes, we will need conditional rendering to not show these in cloud service.
2. We do not have a way to that yet. Misha is coordinating an effort to have a generalized solution for this available sometime during 4.2 development cycle.

I think the added value for self-managed customers justifies adding as-is for now.